components/TokenLink/TokenLink.tsx

```diff
@@ -1,19 +1,22 @@
-﻿import React from "react";
+﻿﻿import React from "react";
 import {useRouter} from "next/router";
 import Link from "next/link";
 
-
-type TokenLinkProps = {children: React.ReactNode; href: string}
+type TokenLinkProps = {children: React.ReactNode; href: string; as: string}
```

Comment:
can we get rid of the `as`? we don't ever seem to use it.
(we can actually also get rid of children - that's handled automatically by the FunctionComponent)

Reply:
yes, this has been overridden by the stepper anyway, I'll delete